src/mui/auth/Logout.js

```diff
@@ -23,9 +23,13 @@ Logout.propTypes = {
     userLogout: PropTypes.func,
 };
 
+const mapStateToProps = state => ({
```

Comment:
This was already fixed in master in #606. Can you restrict your PR to the change in `src/sideEffect/saga/auth.js`?